README.md

```diff
@@ -382,6 +391,21 @@ new event listener whose events are exposed through the Observer handler
 functions and are composable with the various
 [combinators](#operators) available to all Observables.
 
+#### Constructing & converting objects to Observables
+
+Observables can be created by their native constructor, as demonstrated above,
+or by the `Observable.from()` static method. This method constructs a native
+Observable from any of the following objects:
+
+ - `Observable` (in which case it just returns the given object)
```

Comment:
Interesting. `Array.from(array) !== array`, but it looks like `Iterator.from(iterator) === iterator`. Sticking with the most-recent precedent makes sense, but this is a bit surprising to me.

@bakkot do you know if this asymmetry was discussed and chosen consistently? I feel like I've used `Array.from(array)` as a sort of clone operation in the past.

Maybe the main difference is that iterators and observables are both "immutable" (from the consumer's point of view), whereas arrays are mutable?

Reply:
> do you know if this asymmetry was discussed and chosen consistently?

Yes, though not at length. Iterators are pretty different from arrays - mutability, as you point out, but also you can't _actually_ clone an iterator, since they're stateful. As such we didn't feel the `from` method needed to behave identically (since in fact it can't anyway).

The core of the operation is "give me an iterator from this thing", and there's no reason to make an extra object when the thing you've given is already an iterator with the right prototype. There's no particular benefit, so it would just be wasteful.